packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts

```diff
@@ -800,7 +805,13 @@ export default class NotificationServicesController extends BaseController<
 
       const { accounts } = await this.#accounts.listAccounts();
 
-      let userStorage = await this.#getUserStorage();
+      // Attempt Get User Storage
+      // Will be null if entry does not exist, or a user is resetting their notifications
+      // Will be defined if entry exists
+      // Will throw if fails to get the user storage entry
+      let userStorage = opts?.resetNotifications
+        ? null
+        : await this.#getUserStorage();
```

Comment:
This will help resolve corrupted state on mobile, and unblock this mobile flow: 
https://github.com/MetaMask/metamask-mobile/pull/11455